src/org/omegat/gui/glossary/GlossaryTextArea.java

```diff
@@ -451,4 +446,29 @@ public void actionPerformed(ActionEvent e) {
                 .setPreference(Preferences.GLOSSARY_SORT_BY_LENGTH, sortOrderLocLength.isSelected()));
         menu.add(sortOrderLocLength);
     }
+
+    private static JMenuItem createMenuItem(String titleKey, String command, Runnable action) {
```

Comment:
If you plan to use the same methodology for main menu, you should move these methods to a common class, for example org.omegat.gui.common.MenuFactory